lyric_finder/src/lib.rs

```diff
@@ -136,6 +138,60 @@ impl Default for Client {
     }
 }
 
+/// Returns `query` without `remaster` & `remix` information from track/artist query.
+/// Returned value is lowercase.
+/// These caused wildly invalid lyrics to be found.
+/// (try yourself adding remastered 2011 to a song's name when searching in Genius!)
+fn improve_query(query: &str) -> String {
+    let is_dash = |c: char| c == '-' || c == '—' || c == '–';
```

Comment:
this is interesting, doesn't Spotify only use `-` as the dash in most songs? Looking into the codes, it's hard to see the differences between these three cases. IMO, it should be fine to check with `-` only.

Reply:
Did it just in case. Will change to your preference :)